pkg/apis/policy/v1alpha1/well_known_constants.go

```diff
@@ -1,12 +1,27 @@
 package v1alpha1
 
 const (
+	// PropagationPolicyReferenceUID is the uid of PropagationPolicy object.
+	PropagationPolicyReferenceUID = "propagationpolicy.karmada.io/uid"
```

Comment:
```suggestion
	// PropagationPolicyUIDLabel is the uid of PropagationPolicy object.
	PropagationPolicyUIDLabel = "propagationpolicy.karmada.io/uid"
```

In addition, please group the new-added ones together, it will make things easy when we are going to deprecate the others.